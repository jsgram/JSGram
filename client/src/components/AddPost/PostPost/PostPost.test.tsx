import React from 'react';
import { PostPhoto } from './index';
import { shallow } from 'enzyme';

describe('PostPost component', () => {
    let renderer: any;
    const mockCroppedImage = 'some value';
    const mockDescription = 'some value';
    const mockSetDescForPost = jest.fn(() => ('some value'));

    beforeEach(() => {
        renderer = shallow(<PostPhoto
            croppedImage={mockCroppedImage}
            description={mockDescription}
            setDescriptionForPost={mockSetDescForPost}/>);
    });

    test('render - success', () => {
        expect(renderer.html()).toMatchSnapshot();
    });
});
