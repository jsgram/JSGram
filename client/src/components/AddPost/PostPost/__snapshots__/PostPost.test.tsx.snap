// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostPost component render - success 1`] = `"<div class=\\"container\\"><div class=\\"row\\"><div class=\\" mx-auto mt-3 post-label\\">New post</div></div><div class=\\"mt-2 mx-auto post row\\"><img src=\\"some value\\" height=\\"293\\" class=\\"img-fluid rounded w-100\\" alt=\\"cropped img\\"/><textarea name=\\"description\\" placeholder=\\"Write a caption...\\" spellcheck=\\"false\\" maxLength=\\"200\\" class=\\"mt-3 interaction form-control\\">some value</textarea></div></div>"`;
